.github/workflows/ci.yml

```diff
@@ -11,13 +11,14 @@ jobs:
   build:
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         java: 
           - 8
           - 11
           - 14
 
     name: Build and Test
```

Comment:
Consider `name: Build and Test JDK ${{ matrix.java }}, ${{ matrix.os }}`

Reply:
This is already reflected in the github checks. For example `Build and Test / Build and Test (ubuntu-latest, 11) `